README.md

```diff
@@ -7,22 +7,18 @@ The test suite offers four modes of operation:
 
 This is the default. If there are no parameters specified, the test suite runs with embedded H2 database and its driver fetched from Maven central.
 
-## 2. PostgreSQL server in CI
-
-If selected, the test suite uses a resident PostgreSQL server present in the Narayana upstream CI environment. The server is always running, always ready and the test suite does not have any control over it. If you have such server installed, simply overwrite ```pgsql``` prefixed attributes in the pom.xml file, e.g. ```-Dpgsql.servername=pg1.example.com```.
-
-## 3. DBAllocator
+## 2. DBAllocator
 
 When specified, the test suite could take advantage of DBAllocator service and allocate an arbitrary database system.
 It was tested to work well with Postgres family of database systems. The DBAllocator service is not public and thus it is not used in the upstream testing.
 
-## 4. Container - Postgres
+## 3. Container - Postgres
 
 The fourth **and the most versatile mode** is to control a Docker daemon and to pull database images and to start 
```

Comment:
@marcosgopen please can you change "fourth" to "third"?

Reply:
Thanks Tom, nicely spotted. I will update it.